go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/osmosis-labs/osmosis/v7
 go 1.17
 
 require (
-	github.com/CosmWasm/wasmd v0.22.0
+	github.com/CosmWasm/wasmd v0.23.0
```

Comment:
We replace the wasmd import due to this commit for our IAVL changes: https://github.com/osmosis-labs/wasmd/commit/fced0452df2f789a871da41e4d97e62998f51a7c

(Really silly API change)

Should we do this for v0.23.0 instead of v0.22.0?

Reply:
Good to know. I didn't dig into the replace blocks, but was surprised go.sum didn't update.

There is no different in x/wasm between 0.22 and 0.23, it is in app, which you don't import. One of those bugfix version bumps.

In general nice to track the latest, but not urgent.

Reply:
Thanks for the info! Going ahead and merging this then